examples/audits.js

```diff
@@ -0,0 +1,18 @@
+// file: example/audits.js
```

Comment:
Change `example` to `examples` or just remove this comment ;)

Reply:
Good catch, will do

Reply:
Done